src/main/java/org/mongen/core/models/StreetSituation.java

```diff
@@ -0,0 +1,98 @@
+package org.mongen.core.models;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Table(name="street_situation")
+public class StreetSituation implements Serializable{
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+    private boolean living_in_street;
+    private Date start_date;
+    private Date end_date;
+	@Column(updatable=false)
+	private Date created;
+	private Date updated;
+
+	@ManyToMany
+    @JoinTable(
+        name = "street_situation_street_locations",
+        joinColumns = @JoinColumn(name = "street_situation_id"),
+        inverseJoinColumns = @JoinColumn(name = "street_location_id")
+    )
+    private List<Beneficiary> street_locations;
+
+	public StreetSituation() {
```

Comment:
nothing here yet?

Reply:
In which sense? that's only the constructor, or maybe I didn't understand your question